src/pages/AccountDetails.vue

```diff
@@ -0,0 +1,563 @@
+<script setup lang="ts">
+import { useRouter } from 'vue-router';
+import { computed, onMounted, reactive, ref, watch } from 'vue';
+import * as http from '@/shared/api';
+import BaseContentBlock from '@/shared/ui/components/BaseContentBlock.vue';
+import DataField from '@/shared/ui/components/DataField.vue';
+import { useTable } from '@/shared/lib/table';
+import BaseTable from '@/shared/ui/components/BaseTable.vue';
+import BaseLoading from '@/shared/ui/components/BaseLoading.vue';
+import { useErrorHandlers } from '@/shared/ui/composables/useErrorHandlers';
+import type { Account } from '@/shared/api/schemas';
+import { AccountIdSchema } from '@/shared/api/schemas';
+import { parseMetadata } from '@/shared/ui/utils/json';
+import BaseTabs from '@/shared/ui/components/BaseTabs.vue';
+import type { TabAccountTransactions } from '@/features/filter-transactions/model';
+import { ACCOUNT_TRANSACTIONS_OPTIONS } from '@/features/filter-transactions/model';
+import { accountInstructionsAdaptiveOptions } from '@/features/filter-transactions/adaptive-options';
+import {
+  type filterTransactionsModel as ftm,
+  InstructionTypeFilter,
+  TransactionStatusFilter,
+} from '@/features/filter-transactions';
+import TransactionsTable from '@/shared/ui/components/TransactionsTable.vue';
+import InstructionsTable from '@/shared/ui/components/InstructionsTable.vue';
+import { objectOmit } from '@vueuse/shared';
+import BaseLink from '@/shared/ui/components/BaseLink.vue';
+
+const router = useRouter();
+const { handleUnknownError } = useErrorHandlers();
+
+const accountId = computed(() => {
+  const id = router.currentRoute.value.params['id'];
+
+  return AccountIdSchema.parse(id);
+});
+
+const account = ref<Account | null>(null);
+const isFetchingAccount = ref(false);
+
+onMounted(async () => {
+  try {
+    isFetchingAccount.value = true;
+    account.value = await http.fetchAccount(accountId.value);
+
+    listState.authority = accountId.value.toString();
+
+    if (account.value) {
+      const ownedEntities = [];
+
+      if (account.value.owned_assets) ownedEntities.push(assetsTable.fetch({ owned_by: accountId.value.toString() }));
+      if (account.value.owned_domains) ownedEntities.push(domainsTable.fetch({ owned_by: accountId.value.toString() }));
+
+      if (!ownedEntities.length) return;
+
+      await Promise.all(ownedEntities);
```

Comment:
Looks like premature optimisation (which is the root of all evil in development) - no need to care about this at this stage of development. Just load everything. (in future, no need to edit this now)